SU2_CFD/src/iteration_structure.cpp

```diff
@@ -2443,7 +2439,7 @@ void CDiscAdjFluidIteration::SetDependencies(CSolver *****solver,
   }
 
   /*--- Compute coupling between flow and turbulent equations ---*/
-
+  solver[iZone][iInst][MESH_0][FLOW_SOL]->Preprocessing(geometry[iZone][iInst][MESH_0], solver[iZone][iInst][MESH_0], config[iZone], MESH_0, NO_RK_ITER, RUNTIME_FLOW_SYS, true);
```

Comment:
@oleburghardt this call is duplicated, can one of them be removed?

Reply:
I'll check and eventually delete in in #798 Thanks!